server/models/Notification.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  type: { type: String, required: true },
+  description: { type: String, required: true },
+  read: { type: Boolean, default: false },
+  // recipient: [{type: mongoose.Types.ObjectId, ref:"User"}]
```

Comment:
This should be integrated with the controllers, can you uncomment it and do that?